code/modules/cargo/bounties/botany.dm

```diff
@@ -3,12 +3,15 @@
 	var/datum/bounty/item/botany/multiplier = 0 //adds bonus reward money; increased for higher tier or rare mutations
 	var/datum/bounty/item/botany/bonus_desc //for adding extra flavor text to bounty descriptions
 	var/datum/bounty/item/botany/foodtype = "meal" //same here
+	var/datum/bounty/item/botany/format_exception = FALSE //Set to true if the bounty uses a custom format from the one below
```

Comment:
This shouldn't need this variable, just override the reward and desc on those specific bounties.

Reply:
Never mind I guess

Reply:
Yep that was the first thing I tried, but no dice. The New() code overwrote anything I did.